kalite/distributed/static/js/distributed/videoplayer.js

```diff
@@ -1,504 +0,0 @@
-    window.VideoPlayerState = {
```

Comment:
Would be nice to keep blame history when moving/splitting files. I'm still unclear on what's required to make that happen, but it sounds from this like it might be automatic? http://stackoverflow.com/questions/1728922/how-does-git-track-source-code-moved-between-files